services/api/utils.go

```diff
@@ -25,6 +25,7 @@ var (
 	ErrPayloadMismatch      = errors.New("beacon-block and payload version mismatch")
 	ErrHeaderHTRMismatch    = errors.New("beacon-block and payload header mismatch")
 	ErrBlobMismatch         = errors.New("beacon-block and payload blob contents mismatch")
+  ErrNotAcceptable        = errors.New("not acceptable")
```

Comment:
indentation?

Reply:
Probably my fault when fixing merge conflicts. I don't see an issue. Will fix in an editor.

<img width="587" alt="image" src="https://github.com/user-attachments/assets/543ab207-1fec-43ff-b5d3-6c9e5a36f224" />